docs/index.md

```diff
@@ -3,12 +3,62 @@
 page_title: "redpanda Provider"
 subcategory: ""
 description: |-
-  Redpanda Data terraform provider
+  The Redpanda Data Terraform provider is used to manage Redpanda Dedicated and Cloud clusters and Kafka resources within them. To use the provider to connect to a Redpanda Cloud cluster, you need to provide a client_id and client_secret.
+  Example Usage
+  This example demonstrates how to use the Redpanda Terraform provider with various fields populated with dummy values. Ensure you replace these values with your actual data.
+  ```hcl
+  provider "redpanda" {
+    clientid      = "yourclientid"
+    clientsecret  = "yourclientsecret"
+    cloud_provider = "AWS" # Supported values: GCP, AWS
+    region         = "us-west-2"
+    zones          = ["us-west-2a", "us-west-2b", "us-west-2c"]
+  }
+  Define a Redpanda cluster resource
+  resource "redpandacluster" "examplecluster" {
+    name           = "example-cluster"
+    cloudprovider = "AWS"
```

Comment:
The cloud provider should be lowercase: https://github.com/redpanda-data/terraform-provider-redpanda/blob/main/redpanda/resources/network/resource_network.go#L132 because we receive lowercase back from the API.

btw, we don't have this validation in the cluster resource.

Reply:
Also, it's `cloud_provider` https://github.com/redpanda-data/terraform-provider-redpanda/blob/main/redpanda/models/cluster.go#L27

Reply:
yeah sorry autogenerated the content for this one and didn't review as i'm gonna redo it anyway

Reply:
just want to make sure the formatting looks right before going big